airbyte-integrations/connectors/source-google-adwords-singer/source_google_adwords_singer/spec.json

```diff
@@ -10,7 +10,8 @@
       "oauth_client_secret",
       "refresh_token",
       "start_date",
-      "customer_ids"
+      "customer_ids",
+      "conversion_window_days"
```

Comment:
this should not be a required parameter for backwards compatibility

Reply:
Backward compatibility apart, should it be required?

Reply:
nope. It should be optional. It basically decides the lookback window for getting reports. By default this is 30 days, but it should be configurable by the user

Reply:
Yeah I will remove from required